packages/sdk/src/infrastructure/solana-utils/payment/models/PaymentResponse.ts

```diff
@@ -1,6 +1,9 @@
+import { ContentResponse } from '@heliofi/common';
+
 export interface BasePaymentResponse {
   transactionSignature: string;
   swapTransactionSignature?: string;
+  content?: ContentResponse;
```

Comment:
this was the same mistake on our dash, please use the return type from common package for BasePaymentResponse, you can check in develop on dash how it looks now